src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx

```diff
@@ -117,6 +117,11 @@ const LoadForecast: React.FunctionComponent<LoadForecastProps> = ({
  */
 export const forecastToStandardNames: { [key: string]: Set<string> } = {
   air_temperature: new Set(["air_temperature"]),
+  significant_wave_height: new Set([
```

Comment:
Codacy found an issue: [Identifier 'significant_wave_height' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6262731)